src/sentry/api/endpoints/project_metrics.py

```diff
@@ -43,9 +77,6 @@ class ProjectMetricsDataEndpoint(ProjectEndpoint):
 
     def get(self, request, project):
 
-        if not features.has("organizations:metrics", project.organization, actor=request.user):
```

Comment:
Don't you want this check to keep users from orgs that don't have access to metrics out of the endpoints?

Reply:
@markstory Yes, thank you for noticing! Apparently I dropped these checks in a merge-fest and failed to restore them afterwards. Added them now.